src/Compilers/CSharp/Test/Syntax/Parsing/DeclarationExpressionTests.cs

```diff
@@ -134,45 +135,37 @@ public void NullableTypeTest_02()
             EOF();
         }
 
-        [Fact]
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/72720")]
         public void NullableTypeTest_03()
         {
-            UsingStatement("if (e is int? x) {}",
-                // (1,16): error CS1003: Syntax error, ':' expected
-                // if (e is int? x) {}
-                Diagnostic(ErrorCode.ERR_SyntaxError, ")").WithArguments(":").WithLocation(1, 16),
-                // (1,16): error CS1525: Invalid expression term ')'
-                // if (e is int? x) {}
-                Diagnostic(ErrorCode.ERR_InvalidExprTerm, ")").WithArguments(")").WithLocation(1, 16)
-                );
+            UsingStatement("if (e is int? x) {}");
```

Comment:
what actually adds the error now?

also, can you test `if (e is int ? x : y)`?

Reply:
> what actually adds the error now?

There is a binding error `CS8116: It is not legal to use nullable type '<type>?' in a pattern; use the underlying type '<type>' instead`. These tests assert only syntax errors and there are now none of them, which is nice

Reply:
Test added